src/main.c

```diff
@@ -470,6 +470,11 @@ do_request(const struct rustls_client_config *client_config,
   return ret;
 }
 
+enum rustls_result
+verify(__attribute((unused)) void *userdata, __attribute((unused)) const rustls_verify_server_cert_params *params) {
```

Comment:
Since this program is the integration test for `crustls`, perhaps this should verify that the `userdata` and `params` that get passed to the C into Rust and back to C don't get mangled somehow.